project_event/models/project_task.py

```diff
@@ -205,6 +205,16 @@ class Task(models.Model):
         string='Is Created From Template',
         default=False,
     )
+    spectators = fields.Char(
+        string='Spectators',
+     )
+
+    @api.onchange('spectators')
+    def check_numeric(self):
+        try:
+            int(self.spectators)
+        except Exception as exception:
+            raise ValidationError('The Input value should be numeric')
```

Comment:
raise ValidationError(
                        _(
                            'For Translations'
                        ) )